src/ICT4Events/Views/MaterialSystem/Forms/MaterialSystem.cs

```diff
@@ -0,0 +1,383 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using SharedModels.Data.ContextInterfaces;
+using SharedModels.Data.OracleContexts;
+using SharedModels.Models;
+
+
+namespace ICT4Events.Views.MaterialSystem.Forms
+{
+    public partial class MaterialSystem : Form
+    {
+        #region Local Variables
+        private readonly Event _event;
+        private readonly IGuestContext _contextGuest;
+        private readonly IMaterialContext _contextMaterial;
+        private readonly IMaterialTypeContext _contextMaterialType;
+        private bool _rfidScanned;
+        #endregion
```

Comment:
Deze context instances horen thuis in de logic classes, dus verwacht ik eerder private UserLogic _userRepo oid